sunpy/net/helioviewer.py

```diff
@@ -20,46 +20,66 @@
 
 __all__ = ['HelioviewerClient']
 
-
 class HelioviewerClient(object):
     """Helioviewer.org Client"""
-    def __init__(self, url="https://legacy.helioviewer.org/api/"):
+    def __init__(self, url="https://api.helioviewer.org/"):
         """
-        url : location of the Helioviewer API.  The default location points to
-            version 1 of the API.  Version 1 of the Helioviewer API is
-            currently planned to be supported until the end of April 2017.
+        url : str
+            URL that points to the Helioviewer API.
         """
+        self.datasource_info = dict()
         self._api = url
-
+        self.init_src_dict()
+    
+    def init_src_dict(self):
```

Comment:
This should probably be a hidden (`_`) method.

Reply:
Ok, I will change that